### ISSUE TITLE
Deprecation Warning with Pandas

### ISSUE THREAD

**Issue by pbitzer** [project-mjolnir/sindri#14]
When we upgrade to Pandas 1.3+, we get a deprecation warning on `error_bad_lines`. This was used in this [fix](https://github.com/project-mjolnir/sindri/commit/67c98926850ae91637f84f84c976b26a787399eb).

Looks like we now need to use `on_bad_lines='skip` instead. 

(Pandas 1.3+ is happening on current builds, so this should be tested and fixed soon.)

**Comment by CAM-Gerlach** [comment]
Since `on_bad_lines` was only added in Pandas 1.3.0, released a few months ago (and thus using it now will break our existing installs running Pandas <1.3.0) and the legacy `error_bad_lines` parameter won't be dropped until Pandas 2.0 which looks to be almost a year-ish away at least, its probably best to defer this change until then, and at least a minor version of Sindri. If it is sufficiently annoying in the logs, we can silence it with 

```python
# Use `on_bad_lines="skip"` instead of `error_bad_lines` for Pandas >=1.3.0
warnings.catch_warnings():
    warnings.filterwarnings(
        action="ignore",
        category=DeprecationWarning,
        module="pandas*"
    status_data = pd.concat(
        (pd.read_csv(file, error_bad_lines=False) for file in files_to_load),
        ignore_index=True, sort=False)
)
```

**Comment by pbitzer** [comment]
It's pretty annoying in the logs, since this occurs a lot. I'm good with deferring, and instead just catching this warning.

**Comment by CAM-Gerlach** [comment]
I've fixed this for good in v0.3.0a1, and will backport the warning suppression to v0.2.11.

**Comment by CAM-Gerlach** [comment]
Fixed by #16
